src/main/include/Diagnostics.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 FRC Team 3512. All Rights Reserved.
+
+#pragma once
+
+#include "NetworkTableUtil.hpp"
+
+namespace frc3512 {
+
+class Diagnostics {
```

Comment:
An entire class isn't needed for this. You could have just one function that makes a NetworkTableEntry, then checks the value in it. Since NT is a handle-based API internally, it'll be cheap.